index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mulinde Derrick Portfolio</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="logo-placeholder">
+          <p class="logo">M.D</p>
+        </div>
+        <nav class="nav-links">
+          <img class="menu-btn" src="/images/menu-btn.svg" alt="Menu Button" />
```

Comment:
- [ ] Kindly check the link to your hamburger icon as the icon doesn't display on the webpage.